pallets/committee/src/lib.rs

```diff
@@ -260,6 +278,33 @@ pub mod pallet {
                 })
             })
         }
+
+        /// Add a new constituent member and returns the storage key value pair
+        ///
+        /// This call only triggers after the approving of `propose_constituent`
+        fn add_constituent_encoded(
```

Comment:
This should be an extrinsic that ensures that it has been dispatched by the committee's origin, and if so adds a new constituent.